src/webapp/vue.config.js

```diff
@@ -27,7 +27,6 @@ module.exports = {
   },
 
   devServer: {
-    index: process.env.MELTANO_EMBED == '1' ? 'index-embed.html' : 'index.html',
```

Comment:
This seems deprecated from what I can tell:

```
 INFO  Starting development server...
 ERROR  ValidationError: Invalid options object. Dev Server has been initialized using an options object that does not match the API schema.
         - options has an unknown property 'index'. These properties are valid:
           object { allowedHosts?, bonjour?, client?, compress?, devMiddleware?, headers?, historyApiFallback?, host?, hot?, http2?, https?, ipc?, liveReload?, magicHtml?, onAfterSetupMiddleware?, onBeforeSetupMiddleware?, onListening?, open?, port?, proxy?, server?, setupExitSignals?, setupMiddlewares?, static?, watchFiles?, webSocketServer? }
```

From the docs / discussions it seems like the `historyFallbackApi` already accomplishes the same thing.